test/passing/module.ml

```diff
@@ -28,3 +28,41 @@ module C = struct
 
   and B : sig end = B
 end
+
+let _ =
+  ( module M : S
+    with type long_long_type = t
+     and type long_long_long_type = u
+     and type very_long_long_type = t )
+
+let _ =
+  ( module struct
+      type t
+    end : S with type t = t )
+
+let _ =
+  ( module struct
+      type t
+    end : S )
+
+let _ =
+  ( module struct
+      type t
+    end
+  : Loooooooooooooooonnnnnnnnnnnnnnnng.Looooooooooooooonnnnnnnnnnnnnng.Mod
```

Comment:
Should the `:` be on the same line as `end` and the closing `)` just after the type ?

Reply:
The `:` is currently in the right place. Due to the length of the name, I think the `)` is in the right place or else it would encroach into the margin.